docs/modules/rust-guide/pages/rust-quickstart.adoc

```diff
@@ -121,43 +82,79 @@ cd rust_hello
 
 == Modify the default configuration
 
-Creating a new project adds a default `+dfx.json+` configuration file to your project directory.
-This file contains settings required to build a project for the {IC} much like the `+Cargo.toml+` file provides build and package management configuration details for Rust programs.
-You need to modify the default settings in both of these files to build Rust programs that run on the {IC} as canisters.
+Creating a new project adds several template files to your project directory much like when you create a new Rust package using the `+cargo new+` command.
+You need to modify these default files and add some Rust-specific files to your project before you can build canisters that run on the {IC}.
 
-=== Delete the default program
+=== Replace the default program
 
-Creating a new project creates a default `+src+` directory with a template `+main.mo+`. We'll eventually replace this with a `+lib.rs+` file. 
+Creating a new project creates a default `+src+` directory with a template `+main.mo+`. We'll replace this default program with a Rust library `+lib.rs+` package and `+Cargo.toml+` files. 
 
-. Delete the `+main.mo+` file. 
+To replace the default program:
 
-. In the terminal, run the following command:
+. In the terminal, check that you are in the root directory for your project, if needed.
+. Delete the template `+main.mo+` file by running the following command:
++
+[source,bash]
+---- 
+rm src/rust_hello/main.mo
+----
+. Create a new cargo package using a library template by running the following command:
 +
 [source, bash]
 ----
 cargo init --lib
 ----
++
+This command creates a `+src+` directory with a library (`+lib.rs+`) package and a `+Cargo.toml+` file in the project directory.
```

Comment:
I think we actually want to run this command **not** in the 'root directory', but instead from the `src/rust_hello` directory (i.e. the rust_hello canister within the larger dfx project root). That way it writes `src/rust_hello/Cargo.toml` `src/rust_hello/src/lib.rs`.

Because step 1 of this says to "check that you are in the root directory of your project", I first read this as running `cargo init --lib` from there. That would write `./Cargo.toml`, `./src/lib.rs` (adjacent to the two canister directories within `./src` instead of *inside of* one of those canister dirs).

Then when I did it that way, it gets confusing later to 'Edit Cargo settings for the project'.

I think we want to end up with two Cargo.toml:

* rust_hello canister Cargo.toml - generated by `cargo init --lib` from within `src/rust_hello`
* project root Cargo.toml: created manually below and just adds 'workspace.members' as including `"src/rust_hello"`. I think then Cargo knows how to use that to find `src/rust_hello/Cargo.toml`.